### DISCUSSION TITLE
dont build 274/282: llvm-sys(build)

### DISCUSSION THREAD

**Discussion by Sienna-ya** [vosen/ZLUDA#159] [Q&A]
`error: failed to run custom build command for `llvm-sys v150.1.2 (C:\Users\user\zluda\ext\llvm-sys.rs)`

Caused by:
  process didn't exit successfully: `C:\Users\user\zluda\target\release\build\llvm-sys-41c3a60456577445\build-script-build` (exit code: 101)
  --- stderr
  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:6 (cmake_policy):
    The OLD behavior for policy CMP0114 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:11 (cmake_policy):
    The OLD behavior for policy CMP0116 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Warning at C:/Users/user/zluda/ext/llvm-project/third-party/benchmark/CMakeLists.txt:308 (message):
    Using std::regex with exceptions disabled is not fully supported

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:6 (cmake_policy):
    The OLD behavior for policy CMP0114 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:11 (cmake_policy):
    The OLD behavior for policy CMP0116 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Warning at C:/Users/user/zluda/ext/llvm-project/third-party/benchmark/CMakeLists.txt:308 (message):
    Using std::regex with exceptions disabled is not fully supported

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:6 (cmake_policy):
    The OLD behavior for policy CMP0114 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:11 (cmake_policy):
    The OLD behavior for policy CMP0116 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Warning at C:/Users/user/zluda/ext/llvm-project/third-party/benchmark/CMakeLists.txt:308 (message):
    Using std::regex with exceptions disabled is not fully supported

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:6 (cmake_policy):
    The OLD behavior for policy CMP0114 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:11 (cmake_policy):
    The OLD behavior for policy CMP0116 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Warning at C:/Users/user/zluda/ext/llvm-project/third-party/benchmark/CMakeLists.txt:308 (message):
    Using std::regex with exceptions disabled is not fully supported

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:6 (cmake_policy):
    The OLD behavior for policy CMP0114 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:11 (cmake_policy):
    The OLD behavior for policy CMP0116 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Warning at C:/Users/user/zluda/ext/llvm-project/third-party/benchmark/CMakeLists.txt:308 (message):
    Using std::regex with exceptions disabled is not fully supported

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:6 (cmake_policy):
    The OLD behavior for policy CMP0114 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Deprecation Warning at C:/Users/user/zluda/ext/llvm-project/cmake/Modules/CMakePolicy.cmake:11 (cmake_policy):
    The OLD behavior for policy CMP0116 will be removed from a future version
    of CMake.

    The cmake-policies(7) manual explains that the OLD behaviors of all
    policies are deprecated and that a policy should be set to OLD only under
    specific short-term circumstances.  Projects should be ported to the NEW
    behavior and not rely on setting a policy to OLD.
  Call Stack (most recent call first):
    CMakeLists.txt:6 (include)

  CMake Warning at C:/Users/user/zluda/ext/llvm-project/third-party/benchmark/CMakeLists.txt:308 (message):
    Using std::regex with exceptions disabled is not fully supported

  thread 'main' panicked at ext\llvm-sys.rs\build.rs:103:10:
  called `Result::unwrap()` on an `Err` value: Os { code: 3, kind: NotFound, message: "系统找不到指定的路径。" }
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
error: process didn't exit successfully: `target\debug\xtask.exe --release` (exit code: 101)`

**Comment by vosen** [comment]
Should be fixed in the latest commit
